eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -31,7 +31,7 @@ function Generate-Apiview-File($packagePath)
     try
     {
         npm install
-        cadl compile . --emit=@azure-tools/cadl-apiview
+        cadl compile . --emit=@azure-tools/cadl-apiview --warn-as-error=false

```

Comment:
Can you help me understand more why warnings cause issues? Or are they errors?

Reply:
They are definitely warnings and output is generated despite the warnings when run locally. But on CI it seems no output is generated, almost as if the CI were treating warnings as errors.